recipe/meta.yaml

```diff
@@ -100,8 +101,9 @@ requirements:
   - swig
   - tnt
   - x264
+  - x265
   - xalan-c
-  - xerces-c=3
+  - xerces-c
```

Comment:
Why is this changed here but not in #4824

Reply:
I will update it, what should happen with these in the future is more or less the same PR should just go into both